deploy/kubernetes/manifests/v1/ksNamespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-system
```

Comment:
I think it's an artefact of something that we didn't have to do the first place, also the way namespaces are created has changed recently also. It's okay keep it, if we have scripts that refer to it, and remove when we move to `kubeadm`...